src/common/page.rs

```diff
@@ -51,13 +51,26 @@ pub struct Chunk {
   pub size: usize, // pub owned: false
 }
 
-
 impl fmt::Display for Chunk {
   fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
     write!(f, "Chunk {{ptr:{}, size:{}}}", self.ptr as usize, self.size)
   }
 }
 
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct Run {
+  pub length: *const u8,
+  pub value: *const u8,
```

Comment:
Thanks for your comment. You mean ```libc::c_void```?
Actually, I followed the implementation of Chunk. Would you share your intention?